Makefile

```diff
@@ -2,14 +2,14 @@ SHELL := /usr/bin/env bash
 MAKEFILES=c21e/Makefile \
 	cucumber-messages/Makefile \
 	gherkin/Makefile \
-	fake-cucumber/Makefile \
 	cucumber-query/Makefile \
 	cucumber-react/Makefile \
 	html-formatter/Makefile \
 	json-formatter/Makefile \
 	datatable/Makefile \
 	config/Makefile \
 	cucumber-expressions/Makefile \
+	fake-cucumber/Makefile \
```

Comment:
The dependency order is now  wrong because cucumber-query’s tests depends on fake-cucumber.

Better to move cucumber-expressions up and leave fake-cucumber where it was.

Reply:
indeed, and the formatters also depend on this library. I fix that in the next commit.